test/powershell/Modules/Microsoft.PowerShell.Security/SecureString.Tests.ps1

```diff
@@ -1,5 +1,9 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
+
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingConvertToSecureStringWithPlainText', '')]
+param()
+
```

Comment:
Are changes in the file related to the PR?

Reply:
Unfortunately I had to touch that file due to a credscan failure and had to suppress that rule so that CI passes